phidgets_high_speed_encoder/src/high_speed_encoder_ros_i.cpp

```diff
@@ -19,18 +19,18 @@ HighSpeedEncoderRosI::HighSpeedEncoderRosI(ros::NodeHandle nh,
     {
         char str[100];
         sprintf(str, "joint%u_name", i);
-        nh_.getParam(str, joint_names_[i]);
+        nh_private_.getParam(str, joint_names_[i]);
 
         sprintf(str, "joint%u_tick2rad", i);
-        nh_.getParam(str, joint_tick2rad_[i]);
+        nh_private_.getParam(str, joint_tick2rad_[i]);
 
-        ROS_INFO("Channel %u: '%s'='%s'", i, str, joint_names_[i].c_str());
+        ROS_INFO("Channel %u Name: '%s', Tick2rad: '%f'", i, joint_names_[i].c_str(), joint_tick2rad_[i]);
```

Comment:
```suggestion
        ROS_INFO("Channel %u Name: '%s', Tick2rad: '%f'", i,
                 joint_names_[i].c_str(), joint_tick2rad_[i]);
```

This change should fix [this code style test](https://travis-ci.org/github/ros-drivers/phidgets_drivers/jobs/700421992#L1312-L1316).